src/__mocks__/getRandom/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as faker from 'faker';
+
+export const getRandomString = (length?:number): string => {
+  if(typeof length === 'undefined')
+      return faker.datatype.string();
+  else
+    return faker.datatype.string(length);
+};
+
+export const getRandomNumberMin = (min:number ):number => {
+    return faker.datatype.number({min: min});
+};
+
+export const getRandomNumberRange = (min:number, max:number ):number => {
+    return faker.datatype.number({min: min, max: max});
+};
+
+export const getRandomNumber = ():number => {
+    return faker.datatype.number();
+};
+
+export const getRandomNumArray = (length?:number) => {
```

Comment:
A better way to do this is to set a default value for the length parameter when it's not given:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

That way it always runs the same code (it's a bit overkill to have it also randomize the length of the array when specified)